Sources/Sentry/include/SentryFeedbackAPI+Private.h

```diff
@@ -0,0 +1,17 @@
+#import "SentryFeedbackAPI.h"
+
+#if TARGET_OS_IOS && SENTRY_HAS_UIKIT
+
+NS_ASSUME_NONNULL_BEGIN
+
+// Helper category to allow us to create an instance of SentryFeedbackAPI` within the SDK.
+
+@interface SentryFeedbackAPI ()
+
++ (instancetype)newInstance;
```

Comment:
`l`: I don't really see the value in having a static method to use `[SentryFeedbackAPI newInstance]` instead of `[[SentryFeedbackAPI alloc] init]`. It doesn't seem like a pattern we use a lot in the SDK.